tests/fixtures/f90.mnl

```diff
@@ -0,0 +1,7 @@
+&salad
```

Comment:
The name of this file should be `f90.nml` instead of `f90.mnl`.

Reply:
opps, done.

Reply:
oops again, done and pushed